code/script.js

```diff
@@ -1,47 +1,152 @@
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  currentQuestion = 1
+    showMessage(`Hello there, What's your name?`, 'bot')
+}
+const dogQuestion = () => {
+  currentQuestion++
+  setTimeout(() => showMessage('Do you like dogs', 'bot'), 1000)
+    
+    inputWrapper.innerHTML =  `
+    <button id="yes">YES</button>
+    <button id="no">NO</button>
+    `
+ document.getElementById('yes').addEventListener('click', () => {
+    showMessage('Yes', 'user')
+    setTimeout(() => showMessage('That´s what I thought! I mean who doesn’t?!', 'bot'), 500)
+    handleInput()
+  })
+  document.getElementById('no').addEventListener('click', () => {
+    showMessage('No', 'user')
+    setTimeout(() => showMessage('Wrong answer! Who are you?! A cat person?', 'bot'), 1000)
+    setTimeout(() => showMessage('Try another Bot! Bye bye! 👋🏼', 'bot'), 1500)
+    wrongAnswer()
+  })
+} 
+
+const wrongAnswer = () => {
+  currentQuestion++
+    inputWrapper.innerHTML =  `
+    <button id="bye">Bye!</button>
+    ` 
+    document.getElementById('bye').addEventListener('click', () => {
+    location.reload ()    
+})
+}
+  const breedQuestion = () => {
+    currentQuestion++
+    setTimeout(() => showMessage(`What breed do you like?`, 'bot'), 1500)
+      inputWrapper.innerHTML =  `
+      <button id="kaffe">German Shepard</button>
+      <button id="svante">Labrador</button>
+      `
+
+    document.getElementById('kaffe').addEventListener('click', () => {
+      showMessage('German Shepard', 'user')
+      setTimeout(() => showMessage('Yay! That makes us two!', 'bot'), 1000)
+      handleInput()
+    }
+    )
+    document.getElementById('svante').addEventListener('click', () => {
+      showMessage('Labrador', 'user')
+      setTimeout(() => showMessage('Yay! That makes us two!', 'bot'), 1000)
+      handleInput()
+  })
+}
+
+  const byeBye = () => {
+    currentQuestion++
+      setTimeout(() => showMessage('Bye for now 👋🏼', 'bot'), 1500)
+      inputWrapper.innerHTML =  `
+      <button id="bye">Bye bye!</button>
+      `
+      document.getElementById('bye').addEventListener('click', () => {
+        
+      location.reload () 
+}
+)
+  }
+
+//Here we change the currentQuestion varible
+const handleInput = (event) => {
+    
+  if (currentQuestion === 1) {
+    handleName() }
+    else if (currentQuestion === 2){
+    dogQuestion()
+  } else if (currentQuestion === 3) {
+    breedQuestion()
+  } else {
+    byeBye()
+  }
+  currentQuestion++
 }
 
 // Set up your eventlisteners here
+nameForm.addEventListener('submit', (event) => {
+    event.preventDefault ()
+    handleInput()
+    })
+
+  const handleName = () => {
+  const name = nameInput.value
+    showMessage(name, 'user')
+    nameInput.value = ''
+
+    const answerOne = (`Hi ${name}! Welcome!`) 
+    setTimeout(() => showMessage(answerOne, 'bot'), 1000)
+    setTimeout(dogQuestion, 1000)
+  }
+
+voff.addEventListener('click', () => {
```

Comment:
The "play me a voff"- and "give me a mjau!"-buttons are impressive and funny as heck